### PR TITLE
Add python3 pyqt5 package

### PR DESCRIPTION
**Pull request by hellantos**
Please add the following dependency to the rosdep database.

## Package name:

python3-pyqt5

## Package Upstream Source:

https://riverbankcomputing.com/software/pyqt/download

## Purpose of using this:

The goal of this key is to eventually replace python3-qt5-bindings.
This key will install qt python libraries wrapped with PyQt (SIP). The key installs the same libraries as python3-pyside2 see #34243. This is thought to simplify switching between pyqt and pyside2 as python wrapper providers (see https://github.com/ros-visualization/python_qt_binding/issues/114)

It will at least install wrappers for the following qt libraries:

* qtcore
* qtgui
* qthelp
* qtnetwork
* qtprintsupport
* qtsvg
* qttest
* qtwidgets
* qtxml

In addition it will install:

* sip (generator for pyside2 wrappers)
* development libraries

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - https://packages.debian.org/search?suite=default&section=all&arch=any&searchon=names&keywords=python3-pyqt5
  - https://packages.debian.org/search?suite=default&section=all&arch=any&searchon=names&keywords=python3-sip
  - https://packages.debian.org/search?suite=default&section=all&arch=any&searchon=names&keywords=pyqt5-dev
- Ubuntu: https://packages.ubuntu.com/
   - https://packages.ubuntu.com/search?keywords=python3-pyqt5&searchon=names
   - https://packages.ubuntu.com/search?suite=default&section=all&arch=any&keywords=python3-sip&searchon=names
   - https://packages.ubuntu.com/search?suite=default&section=all&arch=any&keywords=pyqt5-dev&searchon=names

Keys for other distributions are copied over from python3-qt5-bindings.

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -8159,6 +8159,24 @@ python3-pyqrcode:
   gentoo: [dev-python/pyqrcode]
   nixos: [python3Packages.pyqrcode]
   ubuntu: [python3-pyqrcode]
+python3-pyqt5:
+  alpine: [py3-qt5]
+  arch: [python-pyqt5]
+  debian:
+    '*': [pyqt5-dev, python3-pyqt5, python3-pyqt5.qtsvg, python3-sip-dev, qtbase5-dev]
```
Comment (tfoote) [inline rosdep/python.yaml:8166]: Why are you using the wildcard notation and calling out jessie and stretch? They appear to be the same keys.

File: rosdep/python.yaml [hunk 2]
```diff
@@ -8159,6 +8159,24 @@ python3-pyqrcode:
...
+    stretch: [pyqt5-dev, python3-pyqt5, python3-pyqt5.qtsvg, python3-sip-dev, qtbase5-dev]
+  fedora: [python3-qt5-devel, python3-sip-devel, libXext-devel]
+  gentoo: [dev-python/PyQt5]
+  nixos: [python3Packages.pyqt5]
+  openembedded: [python3-pyqt5@meta-qt5]
+  opensuse: [python3-qt5]
+  rhel: ['python%{python3_pkgversion}-qt5-devel', 'python%{python3_pkgversion}-sip-devel', libXext-devel, redhat-rpm-config]
+  slackware: [python3-PyQt5]
+  ubuntu:
+    '*': [pyqt5-dev, python3-pyqt5, python3-pyqt5.qtsvg, python3-sip-dev]
```
Comment (tfoote) [inline rosdep/python.yaml:8177]: Similarly why the wildcard here with artful and bionic called out separately with what looks like the same keys as well.

File: rosdep/python.yaml (hunk 1 above)
Comment (ipa-cmh) [reply rosdep/python.yaml:8166]: This was a copy paste artifact. it is now removed.

File: rosdep/python.yaml (hunk 2 above)
Comment (ipa-cmh) [reply rosdep/python.yaml:8175]: This was a copy paste artifact. it is now removed.

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,4 @@
 *~
 .DS_STORE
 *.pyc
+.vscode/
```
Comment (tfoote) [inline .gitignore:4]:
```suggestion
```
This looks like an unrelated change that slipped into the last commit.

**Review by clalancette — CHANGES_REQUESTED** [review]
Needs update from the original contributor.

**Review by jacobperron — APPROVED** [review]

**Review by ivanpauno — APPROVED** [review]
